### ISSUE TITLE
Is RTP over QUIC in scope

### ISSUE THREAD

**Issue by afrind** [afrind/moqbof#3]
The true scope we should be considering here is possibly larger than what the proposal currently contains. Back in "the day" when people wanted to do RTP over QUIC, they were told to go away for a while until QUIC was done and fully rolled out. It might be time to re-engage that larger conversation.

**Comment by SpencerDawkins** [comment]
I'm not sure what the "proposal currently contains" - which proposal are we talking about?

My theory was that RTP over QUIC was, at least potentially, low-handing fruit, and that finding a home for that work would take something off the table that we (at least theoretically) already see a solution to, so sending it off to AVTCORE (as described in in [Section 1.4 of my SDP draft](https://spencerdawkins.github.io/sdp-rtp-quic/draft-dawkins-avtcore-sdp-rtp-quic.html#name-contribution-and-discussion)) seemed reasonable to me, especially after the AVTCORE session in IETF 112. 

Do we need to pull that back into MOQ?

**Comment by afrind** [comment]
> I'm not sure what the "proposal currently contains" - which proposal are we talking about?

I think they may be referring to the two use cases which are scoped in the bof proposal?

> sending it off to AVTCORE ... seemed reasonable to me

Sounds reasonable to me too.

> Do we need to pull that back into MOQ?

I don't think so, unless there are proponents of that approach that feel like they address the use cases in the proposal.  I'd really like to keep a large chunk of the focus of the BoF on those use cases.  There's an engaged group of participants that have felt the gaps so acutely that we've designed and deployed solutions at scale to address them.  We're in agreement though that standardizing the approaches could bring even more benefit to the entire community.  So while there are lots of other use cases for "media over QUIC", I recommend we address them with lower priority.

**Comment by kixelated** [comment]
RTP over QUIC is a very slight improvement over RTP over DTLS, with some added complexity. That might be compelling enough for some, but I think MoQ should focus on the developers who think RTP is inadequate. AVTCORE seems like a good place for maintaining RTP in parallel.

**Comment by afrind** [comment]
I think we've sufficiently answered this question -- RTP over QUIC is in avtcore for now.  There's an open question in MoQ about how much of RTP could potentially be re-used in a new protocol, and this is part of what we'll discuss in the BoF.  

Should I make a PR to the proposal to explicitly note this, or ok to close with no-action?

**Comment by SpencerDawkins** [comment]
@afrind - I think doing a PR would be good - whatever finally happens, we didn't forget to think about this! :relaxed:
